includes/admin/pages.php

```diff
@@ -228,43 +231,183 @@ public function html() {
 							$tab_label = isset( $settings['label'] ) ? $settings['label'] : '';
 							$tab_link  = admin_url( 'edit.php?post_type=calendar&page=simple-calendar_' . $this->page . '&tab=' . $tab_id );
 
-							echo '<a href="' . $tab_link . '" class="nav-tab ' . ( $current_tab == $tab_id ? 'nav-tab-active' : '' ) . '">' . $tab_label . '</a>';
+							echo '<a href="' . $tab_link . '" class=" ml-[100px] text-base font-normal text-sc_grey-100' . ( $current_tab == $tab_id ? 'nav-tab-active ml-[100px] text-base font-normal bg-sc_blue-100 border-b-[3px]  pb-[7px] pr-[7px] pl-[7px] border-b-sc_green-200' : '' ) . '">' . $tab_label . '</a>';
 						}
 
 					}
 
 					do_action( 'simcal_admin_page_' . $this->page . '_tabs' );
 
 					echo '</h2>';
+					echo '<div class="bg-sc_blue-100">';
+
+					echo '<div class=" flex pt-[114px]  " >';
+					
 
 					settings_errors();
 
 					foreach ( $settings_pages as $tab_id => $contents ) {
 
 						if ( $tab_id === $current_tab ) {
 
+							echo '<div class="bg-white p-[30] w-[795px] ml-[102px] h-[372px] ">';
+
 							echo isset( $contents['description'] ) ? '<p>' . $contents['description'] . '</p>' : '';
 
 							do_action( 'simcal_admin_page_' .  $this->page . '_' . $current_tab . '_start' );
 
 							settings_fields( 'simple-calendar_' . $this->page . '_' . $tab_id );
-							do_settings_sections( 'simple-calendar_' . $this->page . '_' . $tab_id );
+							do_settings_sections( 'simple-calendar_' . $this->page . '_' . $tab_id  );
 
 							do_action( 'simcal_admin_page_' .  $this->page . '_' . $current_tab . '_end' );
-
+							
 							$submit = apply_filters( 'simcal_admin_page_' . $this->page . '_' . $current_tab . '_submit', true );
 							if ( true === $submit ) {
 								submit_button();
 							}
+							echo '</div>'; 
+							
 						}
 					}
-				}
+					?>
+					<div class="w-[481px] h-[360px] ml-[25px] bg-white" >
+						<div >					
+							<img class="m-auto mt-[19px]" src="<?php echo $admin_image_about_path . '/top-rated 1.png'; ?>" />							
+						</div>
+						<div class=" mt-[15px] text-center font-semibold text-lg">
+							<Span>Please Rate Us !</Span>
```

Comment:
please use translation functions for text everywhere

Reply:
ok i had used that function.